app/views/layouts/_view_buttons.html.haml

```diff
@@ -4,7 +4,10 @@
   - defered_toolbar_render('view_tb', 'compare_view_tb')
 - elsif @lastaction == "drift"
   - defered_toolbar_render('view_tb', 'drift_view_tb')
+-elsif %w(ems_container).include?(@layout)
+  - defered_toolbar_render('view_tb', 'dashboard_summary_toggle_view_tb')
```

Comment:
I think that maybe this should be an ```if ui_feature(...``` and that entities should have a view_dashboard feature